src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/AnswerSteps.java

```diff
@@ -73,7 +70,11 @@ public void an_existing_question_with_id() throws IOException {
         question.setSubject("foo");
         question.setQuestionText("question text");
         question.setQuestionRoundId(1);
-        question.setOnlineHearingId(1);
+
+        OnlineHearing onlineHearing = new OnlineHearing();
+        onlineHearing.setOnlineHearingId(UUID.fromString("d9248584-4aa5-4cb0-aba6-d2633ad5a375"));
+        question.setOnlineHearing(onlineHearing);
+        
```

Comment:
See above comment about hard coding test data. The AAT setup is broken right now because of the database set up. Once that fix is merged into master, it should be possible to call the OnlineHearingService class to create the online hearing for us, or possibly we could just use the @Sql annotation.